modules/class_pv_forecast.py

```diff
@@ -54,51 +67,54 @@ def __init__(self, filepath=None, url=None, cache_dir='cache', prediction_hours=
             self.load_data_with_caching(url)
 
         if len(self.forecast_data) < self.prediction_hours:
-            raise ValueError(f"Die Vorhersage muss mindestens {self.prediction_hours} Stunden umfassen, aber es wurden nur {len(self.forecast_data)} Stunden vorhergesagt.")
+            raise ValueError(
+                f"Die Vorhersage muss mindestens {self.prediction_hours} Stunden umfassen, aber es wurden nur {len(self.forecast_data)} Stunden vorhergesagt."
+            )
 
     def update_ac_power_measurement(self, date_time=None, ac_power_measurement=None):
         found = False
         input_date_hour = date_time.replace(minute=0, second=0, microsecond=0)
 
         for forecast in self.forecast_data:
-            forecast_date_hour = parser.parse(forecast.date_time).replace(minute=0, second=0, microsecond=0)
+            forecast_date_hour = parser.parse(forecast.date_time).replace(
+                minute=0, second=0, microsecond=0
+            )
             if forecast_date_hour == input_date_hour:
                 forecast.ac_power_measurement = ac_power_measurement
                 found = True
```

Comment:
Local variable `found` is assigned to but never used

Reply:
i guess ruff is not looking for that. i just noticed while looking into the code. to keep the pr clean lets keep it for now